src/web/CMakeLists.txt

```diff
@@ -25,15 +26,19 @@ add_custom_target(npm_depdendencies
 
 
 set(command_control_outdir ${project_SHARE_DIR}/jaiabot/web/command_control/dist/client)
+
 add_custom_command(OUTPUT ${command_control_outdir}/client.js ${command_control_outdir}/favicon.png ${command_control_outdir}/index.html ${command_control_outdir}/manifest.json
-  COMMAND ${CMAKE_CURRENT_SOURCE_DIR}/command_control/build.sh
-  ARGS ${command_control_outdir}
-  COMMENT "Running command_control build.sh"
+  COMMAND webpack
+  ARGS --mode production --config ./webpack.config.js --output-path dist/client --stats errors-only
+  WORKING_DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}/command_control
+  COMMENT "Building command_control"
```

Comment:
Should we make this Building jcc for consistency with the jcc webpack command?

Reply:
I believe I did this, and it ended up in one of the other PRs.  Lemme check...

Reply:
Yep, it's in one of the other PRs.